extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -981,14 +983,23 @@ export class SystemLayer {
 
       const defaultConfig = VIEWER_CONTROL_DEFAULTS[control.name];
 
-      if (!defaultConfig) {
-        return;
+      let element;
+      if (defaultConfig && defaultConfig.selector) {
+        element = scopedQuerySelector(
+          this.getShadowRoot(),
+          defaultConfig.selector
+        );
+      } else {
+        element = this.win_.document.createElement('button');
```

Comment:
can you create a ticket to add an extra `label` option from the player configuration?

Reply:
#30917